tests/migrations/0001_initial.py

```diff
@@ -64,4 +64,27 @@ class Migration(migrations.Migration):
             },
             bases=(models.Model,),
         ),
+        migrations.CreateModel(
```

Comment:
No worries! And I don't know if you caught it or not, but I fixed the code coverage loss (this PR now increases coverage by 1.96%) - @codecov-io's [comment](https://github.com/MarkusH/django-osm-field/pull/9#issuecomment-223791232) is out of date at this point in time.